_posts/2022-11-07-road-to-quarkus-3.adoc

```diff
@@ -0,0 +1,110 @@
+
+## What will be in Quarkus 3?
+
+The current planned list of highlights for Quarkus 3 are:
+
+  - *Hibernate ORM 6* - faster, safer, better (Check out https://www.youtube.com/watch?v=pc6QIwx0EL0[this presentation] to see how awesome it is).
+  - *Jakarta EE 10* API and packages - following the latest evolution of the Java world
+  - *Eclipse MicroProfile 6* - the latest update of all the MP specifications continuing evolving alongside the state of the practice
+  - *HTTP/3* support (aka *Quic*) - the https://www.redhat.com/architect/http3[new HTTP version] fixing the head-of-line blocking problem, using UDP.
+  - *io_uring* - the next generation of asynchronous IO support. This is a complete game changer if you are interested in performance and efficiency (response time, latency…)
+  - *Virtual threads* (aka Loom) and *Structured Concurrency Support* - based on the lessons learned from our https://www.redhat.com/architect/http3[initial integration], we are now looking into expanding and improving the support.
+  - *java.util.concurrent.Flow* - Time to evolve; Flow has been in the JDK since Java 9. Thus we want to move from Reactive Streams to `java.util.concurrent.Flow`.
+  - *Documentation* - Work started on better and more structured documentation following the https://diataxis.fr/[Diataxis] framework principles.
+  - *Tooling for updates* - going to be a new major version, we want to make the migration as smooth as possible. Thus we will provide migration tools.
+  - A new *gRPC server*, more flexible and easier to maintain
+  - A *revamped dev UI*
+  - <Your extension awesome feature/improvements here>
+
+
+## Java Target 
+
+Quarkus 3 continues (like Quarkus 2) to target minimal version of Java 11. The default recommendation for users will be to use Java 17. 
+
+We will continue to revise minimal Java version based on user feedback and uptake of Java 17 in the ecosystem. For now we feel it is of higher value to allow as many Java 11 users as possible to upgrade to Quarkus 3 and use the Jakarta based dependencies. 
+
+## Release Plan
+
+We will continue the monthly cadence release of the 2.x version of Quarkus. Quarkus 3.x preview (alphas, betas) will be released in parallel. This blog marks the start of the first Quarkus 3 Alpha.
+
+It will take more than a month to integrate the change set we want and collect feedback (from users, and extension maintainers...) from the previous release. Thus, 3.x release cadence will be a bit slower than usual.
+
+Thus the intent is to have a time period of some months to allow continuous integration and start releasing a Quarkus Platform with the members that are ready to do and then continuously include more.
+
+Proposed plan:
+
+  - Start doing 3.0.0.Alpha core and platform releases at regular intervals to allow easy integration builds, starting first from 2.13.3 in the coming week
```

Comment:
It says `coming week`. Should we directly announce 3.0.0.Alpha1 or do you want a separate blog post with the Alpha1 release and the highlights of the Alpha1 release?